File: code/2033.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
	long double valor, juros;
	int meses;
	
	while(scanf("%Lf", &valor) != EOF)
	{
		scanf("%Lf%d", &juros, &meses);		
		long double simples  = valor * juros * meses;
		long double composto = valor * powl(1 + juros, meses) - valor;					
		printf("DIFERENÇA DE VALOR = %.2Lf\n", fabs(simples - composto));
		printf("JUROS SIMPLES = %.2Lf\n", simples);
		printf("JUROS COMPOSTO = %.2Lf\n\n", composto);
	}
}

File: code/Matemática/p-level-2/2033.py
/*
Nome:      Juros Sobre o Empréstimo
ID:        2033
Resposta:  Accepted
Linguagem: Python 3 (Python 3.4.3) [+1s]
Tempo:     0.016s
Tamanho:   451 Bytes
Submissao: 06/01/18 16:18:08
*/
# -*- coding: utf-8 -*-

''' URI - 2033  '''
while True:
	try:
		valor = float(input())
	except:
		break

	juros = float(input())
	meses = int(input())
	
	simples = valor+(valor*juros*meses)
	composto = valor
	
	for mes in range(meses):
		composto += composto * juros
	
	print('DIFERENCA DE VALOR = %.2f' % abs(simples-composto))
	print('JUROS SIMPLES = %.2f' % (simples-valor))
	print('JUROS COMPOSTO = %.2f' % (composto-valor))
